### ISSUE TITLE
Issue with k8s.io/docs/reference/access-authn-authz/certificate-signing-requests/

### ISSUE THREAD

**Issue by BaskingWithSharks** [kubernetes/website#35436]
Documentation says
"A CertificateSigningRequest (CSR) resource is used to request that a certificate be signed by a denoted signer".
and
"The CertificateSigningRequest resource type allows a client to ask for an X.509 certificate be issued..."

What is it now ?  "be signed" or "be issued" or do these words denote the same action ?

**Comment by sftim** [comment]
This is mostly a support request I think.

Issuing and signing a certificate are closely related concepts. We could improve the wording, which would help non-native speakers, folks who are new to this, and folks who are in a hurry - amongst others.

I don't see a case for prioritising that cleanup though.
/sig auth
/priority awaiting-more-evidence
/language en

**Comment by sftim** [comment]
Maybe if we link to https://en.wikipedia.org/wiki/Certificate_signing_request for context?

**Comment by divya-mohan0209** [comment]
I agree with Tim that signing and issuing are closely linked and maybe we could word it better + link it to the Wiki link mentioned above.

**Comment by divya-mohan0209** [comment]
I think this change would be worth having in the docs.
/reopen
